dss/index/es/backend.py

```diff
@@ -53,12 +53,17 @@ def remove_bundle(self, bundle: Bundle, tombstone: Tombstone):
         tombstone_doc = BundleTombstoneDocument.from_tombstone(tombstone)
         modified, index_name = doc.entomb(tombstone_doc, dryrun=self.dryrun)
         if self.notify or modified and self.notify is None:
-            self._notify(tombstone_doc, index_name)
+            self._notify_tombstone(doc, tombstone_doc, index_name)
 
     def _notify(self, bundle, index_name):
         subscription_ids = self._find_matching_subscriptions(bundle, index_name)
         self._notify_subscribers(bundle, subscription_ids)
 
+    def _notify_tombstone(self, bundle, bundle_tombstone, index_name):
```

Comment:
```suggestion
    def _notify_tombstone(self, doc, tombstone_doc, index_name):
```

You might disagree with a naming convention but deviating from a convention only adds to the chaos. You'd have to refactor the codebase to your new naming convention.